sympy/printing/lambdarepr.py

```diff
@@ -348,6 +348,13 @@ def _print_Float(self, e):
         args = str(tuple(map(int, e._mpf_)))
         return 'mpf(%s)' % args
 
+    def _print_uppergamma(self,e): #printer for the uppergamma function
+        exp="gammainc({0}, {1}, inf)".format(e.args[0], e.args[1])
+        return "(%s)" %(self._print(exp))
```

Comment:
This combination of two consequitive string formatting operations  (.format and %) could probably be replaced by a single operation with arguments `self._print(e.args[0])` and `self._print(e.args[1])`.